terms.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Terms of Use"
+---
+
+## 1. Introduction
+
+These Terms of Use (the "Terms") govern your use of R-multiverse (the "Repository"). By using the Repository, you agree to comply with these Terms. If you do not agree with these Terms, you may not use the Repository. 
```

Comment:
I'd recommend line breaks after each sentence for better future diff-ability.

Reply:
Thanks for reminding!